packages/material-ui-styles/src/ThemeProvider/ThemeProvider.test.js

```diff
@@ -83,10 +89,10 @@ describe('ThemeProvider', () => {
     }
 
     const wrapper = mount(<Container />);
-    assert.strictEqual(wrapper.text(), 'foobar');
+    assert.strictEqual(text(), 'foobar');
     wrapper.setProps({});
-    assert.strictEqual(wrapper.text(), 'foobar');
-    assert.strictEqual(themes[0], themes[1]);
+    assert.strictEqual(text(), 'foobar');
+    assert.strictEqual(themes.length, 1);
```

Comment:
/cc @oliviertassinari That's a weird one. Will definitely start investigating this.

Reply:
Nevermind. Needs investigation when we don't mock React.memo anymore.